### PR TITLE
Droth 2142 culverts for service points

### PR DESCRIPTION
**Pull request by marquesfm**

### CODE REVIEW

File: UI/src/utils/backend-utils.js [hunk 1]
```diff
@@ -487,10 +483,6 @@
       return $.getJSON('api/suggested/' + municipalityCode + '/' + typeId);
     };
 
-    this.getStructuredInfo = function(municipalityName, municipalityCode) {
-      return $.getJSON('api/buildInfo/' + municipalityName + '/' + municipalityCode);
-    };
-
```
Comment (ricardofmsimoes) [inline UI/src/utils/backend-utils.js]: Are this removes ok?

Comment (marquesfm) [reply UI/src/utils/backend-utils.js]: Attention on this when merge master inside

File: digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/Digiroad2Api.scala [hunk 2]
```diff
@@ -1647,9 +1647,14 @@ class Digiroad2Api(val roadLinkService: RoadLinkService,
...
+    }catch{
+      case _: Exception => None
+    }
+
     val user = userProvider.getCurrentUser()
-    val municipalities: Set[Int] = if(municipalityCode != "null") Set(municipalityCode.toInt) else { if (user.isOperator()) Set() else user.configuration.authorizedMunicipalities }
+    val municipalities: Set[Int] = if(municipalityCode.isDefined) Set(municipalityCode.get) else { if (user.isOperator()) Set() else user.configuration.authorizedMunicipalities }
     municipalityService.getMunicipalitiesNameAndIdByCode(municipalities).sortBy(_.name).map { municipality =>
       Map("id" -> municipality.id,
         "name" -> municipality.name)
```
[the remark this answers is not in the document]
Comment (marquesfm) [reply digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/Digiroad2Api.scala]: Attention on this when merge master inside.

File: digiroad2-oracle/src/test/scala/fi/liikennevirasto/digiroad2/service/pointasset/ServicePointServiceSpec.scala [hunk 3]
```diff
@@ -26,26 +26,38 @@ class ServicePointServiceSpec extends FunSuite with Matchers {
...
+      val id = service.create(IncomingServicePoint(374128.0,6677512.0,Set(IncomingService(6,Some("Testipalvelu1"),Some("Lisätieto1"),Some(3),Some(100), isAuthorityData = true, None),IncomingService(8,Some("Testipalvelu2"),Some("Lisätieto2"),None,Some(200), isAuthorityData = true, None)), Set()),235, "jakke")
       val assets = service.getPersistedAssetsByIds(Set(id))
 
       assets.size should be(1)
 
       val asset = assets.head
       val services = asset.services
-      // TODO: Test service content
+      val firstService =  services.head
+      val secondService =  services.tail.head
```
Comment (marquesfm) [inline digiroad2-oracle/src/test/scala/fi/liikennevirasto/digiroad2/service/pointasset/ServicePointServiceSpec.scala]:
```scala
val secondService =  services.last

Comment (FilPinto) [reply digiroad2-oracle/src/test/scala/fi/liikennevirasto/digiroad2/service/pointasset/ServicePointServiceSpec.scala]: Applied this suggestion

File: UI/src/utils/backend-utils.js (hunk 1 above)
Comment (FilPinto) [reply UI/src/utils/backend-utils.js]: getStructuredInfo is not being used so the remove can be applied

File: digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/Digiroad2Api.scala (hunk 2 above)
[the remark this answers is not in the document]
Comment (FilPinto) [reply digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/Digiroad2Api.scala]: The changes made to this on this branch are not to be applied

**Pull request merged**